tests/unit/data/sboms/cachito_gomod_nodeps.bom.json

```diff
@@ -0,0 +1,39 @@
+{
+  "bomFormat": "CycloneDX",
+  "components": [
+    {
+      "name": "github.com/cachito-testing/cachito-gomod-without-deps",
+      "purl": "pkg:golang/github.com/cachito-testing/cachito-gomod-without-deps@v0.0.0-20200925115855-a888f7261b9a?type=module",
+      "version": "v0.0.0-20200925115855-a888f7261b9a",
+      "properties": [
+        {
+          "name": "cachi2:found_by",
+          "value": "cachi2"
+        }
+      ],
+      "type": "library"
+    },
+    {
+      "name": "github.com/cachito-testing/cachito-gomod-without-deps",
+      "purl": "pkg:golang/github.com/cachito-testing/cachito-gomod-without-deps@v0.0.0-20200925115855-a888f7261b9a?type=package",
+      "version": "v0.0.0-20200925115855-a888f7261b9a",
+      "properties": [
+        {
+          "name": "cachi2:found_by",
+          "value": "cachi2"
+        }
+      ],
+      "type": "library"
+    }
+  ],
+  "metadata": {
+    "tools": [
+      {
+        "vendor": "red hat",
+        "name": "cachi2"
+      }
+    ]
+  },
+  "specVersion": "1.4",
+  "version": 1
+}
```

Comment:
Nitpick: I think that even for these test files, we are adding a trailing empty line.

Reply:
That's how the files were generated by Cachi2, should I add a newline?

Reply:
That is a good point. I looked at the other files in the same dir, and they all have a newline. Nevermind it then, as I said, it's a nitpick.